frontend/src/index.css

```
@@ -47,6 +47,9 @@
     --ring: 164 61% 70%;
     --radius: 0.5rem;
     --loader: #003c57;
+    --statusgreen:#bbf7d0 ;
+    --statusred: #fecaca;
+    --statusyellow:#fef08a ;
```

We can use more descriptive names for these 
```suggestion
    --status-success: #bbf7d0 ;
    --status-error: #fecaca;
    --status-warning: #fef08a ;
```